Modifs dessins Grand Castor

robertxa opened the pull request:
J'ai modifié le dessin du Grand Castor

robertxa committed:
Modif nom du scialet du Grand Castor

robertxa committed:
Merge branch 'BenoitURRUTY:main' into main

robertxa committed:
Correction Grand Castor

robertxa committed:
Merge branch 'main' of https://github.com/robertxa/Synthese-Clot-d-Aspres

robertxa committed:
Correction Apocalypso

robertxa commented:
J'ai ajouté la révision du scialet Apocalypso qui doit maintenant être OK !

BenoitURRUTY commented:
Punaise t'en fais plein! Merci pour ta contribution.


Le mar. 21 janv. 2025 à 15:56, Xavier Robert ***@***.***> a
écrit :

> J'ai ajouté la révision du scialet Apocalypso qui doit maintenant être OK !
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/BenoitURRUTY/Synthese-Clot-d-Aspres/pull/9#issuecomment-2604959190>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AHZYUW3DOXTHJTZXEKZPVYL2LZNYVAVCNFSM6AAAAABVQWJWS6VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDMMBUHE2TSMJZGA>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>

robertxa committed:
Révision Brise de Coeurs

L'aval des Globules Bleus n'est pas satisfaisant... La topo est assez pourrie...

robertxa committed:
Revision CAF 132 et 131

robertxa commented:
Salut,

J'ai ajouté la révision de Brise de Coeur.
L'aval des Globules Bleus est à reprendre, c'est tout pourri (mais bon, les données topos sont assez succinctes...), et il y a 2 réseaux que sont non topographiés !

N'hésite pas à accepter la pull request pour mettre à jour le repo !
